tests/test_utils/test_transactiontestcase.py

```diff
@@ -1,7 +1,27 @@
+
+from django.db import connections
 from django.test import TransactionTestCase, mock
 
+from .models import Car
+
+
+class TestSerializedContentMockMixin(object):
+    initial_data_migration = '[]'
+    _connections_test_serialized_content = {}
+
+    def _pre_setup(self):
```

Comment:
I feel like we shouldn't override "private" methods to test, I also don't see why that would be necessary here.